runners/pom.xml

```diff
@@ -40,6 +40,7 @@
     <module>google-cloud-dataflow-java</module>
     <module>spark</module>
     <module>apex</module>
+    <module>tez</module>
```

Comment:
<!--new_thread; commit:346274d62540a16c9bf795fc6e9ab2cd28430008; resolved:0-->
This should actually be in the Java 8 profile. If you rebase on master right now, you'll see that the Gearpump runner also requires Java 8 and has a profile set up for it.